infra/base-images/base-builder/Dockerfile

```diff
@@ -84,9 +84,11 @@ RUN rustup component add rust-src --toolchain nightly
 # Default build flags for various sanitizers.
 ENV SANITIZER_FLAGS_address "-fsanitize=address -fsanitize-address-use-after-scope"
 
-# Set of '-fsanitize' flags matches '-fno-sanitize-recover' + 'unsigned-integer-overflow'.
```

Comment:
could you please add this line back? I'm worried that otherwise one day someone may accidentally notice the mismatch around `unsigned-integer-overflow` and try to fix it, although it's intentional